cmd/config/list_keys.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"github.com/pingidentity/pingcli/cmd/common"
+	config_internal "github.com/pingidentity/pingcli/internal/commands/config"
+	"github.com/pingidentity/pingcli/internal/configuration/options"
+	"github.com/pingidentity/pingcli/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+const (
+	listKeysCommandExamples = `  List all configuration keys stored in the CLI configuration file.
```

Comment:
Probably be good to have both `pingcli config list-keys` and `pingcli config list-keys --yaml` described